job_spec/INSAR_ISCE_BURST.yml

```
@@ -1,23 +1,38 @@
 INSAR_ISCE_BURST:
   required_parameters:
-    - granules
+    - reference
+    - secondary
   parameters:
-    granules:
+    reference:
       default:  '""'
       api_schema:
         type: array
-        minItems: 2
-        maxItems: 2
+        minItems: 1
+        maxItems: 1
         example:
           - S1_136231_IW2_20200604T022312_VV_7C85-BURST
-          - S1_136231_IW2_20200616T022313_VV_5D11-BURST
         items:
-          description: Name of the Sentinel-1 SLC IW burst granule to process
+          description: Name of the reference Sentinel-1 SLC IW burst granule to process
           type: string
-          pattern: "^S1_\\d{6}_IW.*-BURST"
+          pattern: '^((S1_\d{6}_IW\d_\d{8}T\d{6}_[VH]{2}_([\dA-F]){4}-BURST)($|[ \t]+))+$'
```

No longer using a space-separated list.
```suggestion
          pattern: '^S1_\d{6}_IW\d_\d{8}T\d{6}_[VH]{2}_([\dA-F]){4}-BURST$'
```